### ISSUE TITLE
Non-admin user can't change password

### ISSUE THREAD

**Issue by yarg-kane** [traggo/server#71]
While trying to setup a few accounts for friends, I noticed that the admin users can change everybody's password (normal), but the normal users can't manage their own at all.

**Comment by jmattheis** [comment]
Good catch. A change password form should be added on the settings page.

**Comment by ethanholz** [comment]
Has a change been made to allow for this? I am currently running on v0.2.3@f31968c1 and I cannot see this option for users.

**Comment by jmattheis** [comment]
No, this ticket is still open.
